ynput/OpenPype
# Resolution settings referenced from DB record for 3dsMax

## Changelog Description
1. Add Callback for setting the resolution according to DB after the new scene is created.
2. Add a new Action into openpype menu which allows the user to reset the resolution in 3dsMax

![image](https://user-images.githubusercontent.com/64118225/225937352-dfd24b75-062a-4280-901b-187ac7453365.png)

## Additional info
N/A

## Testing notes:
1. launch 3dsMax via Openpype
2. check the resolution in render setting
4. it should be set according to DB.
Reset Resolution
1. locate "Set Resolution" in openpype menu
2. click it
3. check the resolution in render setting to see if it is same with the setting shown in DB
